### ISSUE TITLE
Add com.apple/notification_event/jsonschema/1-0-0

### ISSUE THREAD

**Issue by mhadam** [snowplow/iglu-central#723]
The schema will be used to implement: snowplow/snowplow-objc-tracker#335

**Comment by mhadam** [comment]
Here's a link to the information available from a push notification:
[Payload Key Reference](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/PayloadKeyReference.html)

**Comment by alexanderdean** [comment]
Thanks @mhadam - please open a PR and assign to @chuwy and me to review

**Comment by mhadam** [comment]
I found a discrepancy in the way that notifications are created and accessed. [`UserNotifications`](https://developer.apple.com/documentation/usernotifications) is the newest framework for delivering notifications in iOS11. It handles both local and remote notifications in one object - you end up with a [`UNNotificationContent`](https://developer.apple.com/documentation/usernotifications/unnotificationcontent). Since that object is the one developers end up interacting with, it would make sense to use that as a basis for the schema rather than the remote notification payload. The remote notification payload is included in the `userInfo` attribute anyway.

I've made a commit with a schema that conforms to the object.
